host/xtest/xtest_6000.c

```diff
@@ -1352,6 +1352,117 @@ static void xtest_tee_test_6014_single(ADBG_Case_t *c, uint32_t storage_id)
 
 DEFINE_TEST_MULTIPLE_STORAGE_IDS(xtest_tee_test_6014)
 
+#ifndef __ANDROID__
```

Comment:
I think this will actually just work on Android. I can try it a bit later. The `/data/tee` path is an Androidism anyway, and on non-Android, we really should be using something like `/var/lib/tee`. But that's a different issue to fix. At a minimum, it'd be nice to figure out how to define the data path in a way that could be shared between the supplicant and the tests, however.